packages/app/src/SessionSubpanel.tsx

```diff
@@ -0,0 +1,637 @@
+import { memo, useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import cx from 'classnames';
+import throttle from 'lodash/throttle';
+import { parseAsInteger, useQueryState } from 'nuqs';
+import ReactDOM from 'react-dom';
+import {
+  ChartConfigWithOptDateRange,
+  DateRange,
+  TSource,
+} from '@hyperdx/common-utils/dist/types';
+import {
+  ActionIcon,
+  Button,
+  Divider,
+  Group,
+  SegmentedControl,
+  Tooltip,
+} from '@mantine/core';
+
+import DOMPlayer from './DOMPlayer';
+import Playbar from './Playbar';
+import SearchInput from './SearchInput';
+import { SessionEventList } from './SessionEventList';
+import { FormatTime } from './useFormatTime';
+import { formatmmss, useLocalStorage, usePrevious } from './utils';
+
+import styles from '../styles/SessionSubpanelV2.module.scss';
+
+const MemoPlaybar = memo(Playbar);
+
+export default function SessionSubpanel({
+  traceSource,
+  sessionSource,
+  onPropertyAddClick,
+  generateChartUrl,
+  generateSearchUrl,
+  setDrawerOpen,
+  rumSessionId,
+  start,
+  end,
+  initialTs,
+}: {
+  traceSource: TSource;
+  sessionSource: TSource;
+  generateSearchUrl: (query?: string, timeRange?: [Date, Date]) => string;
+  generateChartUrl: (config: {
+    aggFn: string;
+    field: string;
+    groupBy: string[];
+  }) => string;
+
+  onPropertyAddClick?: (name: string, value: string) => void;
+  setDrawerOpen: (open: boolean) => void;
+  rumSessionId: string;
+  start: Date;
+  end: Date;
+  initialTs?: number;
+}) {
+  const [selectedLog, setSelectedLog] = useState<
+    | {
+        id: string;
+        sortKey: string;
+      }
+    | undefined
+  >(undefined);
+
+  // Without portaling the nested drawer close overlay will not render properly
+  const containerRef = useRef<HTMLDivElement | null>(null);
+  useEffect(() => {
+    containerRef.current = document.createElement('div');
+
+    if (containerRef.current) {
+      document.body.appendChild(containerRef.current);
+    }
+
+    return () => {
+      if (containerRef.current) {
+        document.body.removeChild(containerRef.current);
+      }
+    };
+  }, []);
+
+  const portaledPanel = null;
+  // TODO: restore this
```

Comment:
since the `LogSidePanel` no longer exists, we need to restore this one in the coming PR